pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -183,7 +183,9 @@ func NewManager(policyName string, machineInfo *cadvisorapi.MachineInfo, nodeAll
 
 // Start starts the memory manager under the kubelet and calls policy start
 func (m *manager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesReady, podStatusProvider status.PodStatusProvider, containerRuntime runtimeService, initialContainers containermap.ContainerMap) error {
-	klog.InfoS("Starting memorymanager", "policy", m.policy.Name())
+	ctx := context.Background()
```

Comment:
```suggestion
	ctx := context.TODO()
```

Reply:
Why not to pass context into this function?

Reply:
Will be exploring passing context as per your suggestion shortly.